airgun/views/trend.py

```diff
@@ -0,0 +1,62 @@
+from widgetastic.widget import Table, Text, TextInput
+from widgetastic_patternfly import BreadCrumb
+
+from airgun.views.common import BaseLoggedInView
+from airgun.widgets import ActionsDropdown, FilteredDropdown
+
+
+class TrendsView(BaseLoggedInView):
+    title = Text("//h1[text()='Trends']")
+    new = Text("//a[contains(@href, '/trends/new')]")
+    table = Table(
+        './/table',
+        column_widgets={
+            'Name': Text('./a'),
+            'Action': ActionsDropdown("./div[contains(@class, 'btn-group')]"),
```

Comment:
I think above would match the two elements as below are classes contain the class `btn-group` in it:
```
class="btn-group"
class="dropup pagination-pf-pagesize btn-group"
```
Can we use below one it provides the uniq element:
```suggestion
            'Action': ActionsDropdown("./div[@class='btn-group']"),
```

Reply:
nope, it will not. In `./div...` we have `.` before `/` that means we have relative path for current ROOT which is `./table`, so we looking inside Table element only

Reply:
ahh , right :)